internals/compiler.rst

```diff
@@ -532,12 +535,12 @@ Important Files
 
     asdl_c.py
```

Comment:
All the files in this section should be linkified.  I left a comment about it in the PR.